tools/wasp-cli/chain/flag.go

```diff
@@ -19,7 +19,11 @@ func defaultChainFallback(chainName string) string {
 }
 
 func getDefaultChain() string {
-	chainSettings := viper.Sub("chains").AllSettings()
+	configSubTree := viper.Sub("chains")
+	if configSubTree == nil {
+		log.Fatal("`chains` field or config file doesn't exist")
+	}
+	chainSettings := configSubTree.AllSettings()
```

Comment:
```suggestion
    chainSettings := map[string]interface{}{}
	if viper.Sub("chains") != nil {
		chainSettings = viper.Sub("chains").AllSettings()
	}
```

This way we fall back nicely into the `case 0` below. The error message there is good enough.